types/index.d.ts

```diff
@@ -4296,7 +4298,8 @@ export interface Help {
 }
 
 export interface ConfigOptions {
-  epsilon?: number
+  relTol?: number
+  absTol?: number
```

Comment:
I think we need to keep `epsilon` here and mark it as deprecated, else existing TypeScript code that uses `epsilon` will break (and the backward compatibility doesn't work).

Reply:
Ok, just did.

How do I mark it as deprecated?

Reply:
Like this:

```ts
/**
 * @deprecated Use `relTol` and `absTol` instead
 */
epsilon?: number
```

Reply:
Thanks!
It's done now.